kong/runloop/plugin_servers/init.lua

```diff
@@ -329,16 +340,13 @@ end
 
 
 function plugin_servers.start()
-  if worker_id() ~= 0 then
-    kong.log.notice("only worker #0 can manage")
-    return
-  end
-
   local pluginserver_timer = proc_mgmt.pluginserver_timer
+  local pluginserver_connection_check_timer = proc_mgmt.connection_check_timer
```

Comment:
We can remove `pluginserver_timer` and `pluginserver_connection_check_timer`,
because both of them will be set into var `handler`.